en/docs/mi-analytics/mi-custom-analytics.md

```diff
@@ -0,0 +1,109 @@
+# Publishing Custom Analytics Events Data
+
+## Introduction
+
+WSO2 Micro Integrator provides the capability to publish custom analytics data with the existing event schema. This guide explains the steps required to achieve this.
+
+This section outlines the process of creating a sample and configuring it with WSO2 Micro Integrator.
+
+## Creating a Sample
+
+To begin, create a new Java/Maven project.
+
+Alternatively, you can use an existing [sample](https://github.com/wso2/product-micro-integrator/tree/master/samples/AnalyticsSampleDataProvider). If you choose to use the provided sample instead of developing one from scratch, you can skip the steps for creating the sample and proceed directly to [Configuring the Sample](#configuring-the-sample).
```

Comment:
Let's add this as a note

Reply:
<img width="1002" alt="image" src="https://github.com/user-attachments/assets/8050220e-9b2e-4ff8-a7d2-ccace81d8ef0" />

Fixed with 951d0756